### PR TITLE
Breaking change: proper getUpdates() support

### PR DESCRIPTION
**Pull request by martintajur**
* Previously, when one fetched deals and then did `deal.getUpdates()` on a specific deal object, it triggered an API request to /v1/deals/:id/updates which is a legacy endpoint, not supported and actually meant to go to full deprecation soon. Instead, the proper endpoint is now used, which is /v1/deals/:id/flow. An exception was needed to the interpretation of the API object structure blueprint.
 * **tl;dr:** /item/:id/flow used instead of /item/:id/updates endpoint is now used when fetching data with `.getUpdates()` method.

### CODE REVIEW

**Comment by martintajur** [comment]
If this were to be released then likely under `4.x` version series.
